src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/models/responses/GameResponse.kt

```diff
@@ -0,0 +1,7 @@
+package com.gamingbacklog.api.gamingbacklogapi.models.responses
+
+class GameResponse(
+        var id: String,
+        var name: String,
```

Comment:
now that the custom stuff has been merged in, we can add all those fields as optional fields too. We should add everything in GameInstance (and put the game instance specific fields with ? since they're optional)